acceptance/localcluster/localcluster.go

```diff
@@ -254,6 +257,10 @@ func (l *Cluster) initCluster() {
 	}
 
 	binds := []string{l.CertsDir + ":/certs"}
+	if (logDirectory != nil) && (len(*logDirectory) > 0) {
+		l.LogDir = filepath.Clean(pwd + "/" + *logDirectory)
```

Comment:
We're computing `pwd` differently on line 249. Should probably unify the computation.